kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -193,9 +193,9 @@ resources:
    user_guildmate_logoff = "~IYour guildmate %s has just departed."
 
    user_safety_on = \
-      "### Your safety is now ~I~gON~n:  You can no longer strike innocents."
+      "### ~B`gSAFETY ON   `nYou can no longer strike innocents."
```

Comment:
Changing italic to bold is a good idea.  I don't know that the language change is an improvement, though.  If I'm a new user and one of the first things I read is "SAFETY ON" in bold green text, I might think I can't be killed.  "Your safety is now ON" certainly isn't super clear either, but it's arguably better.

Reply:
Good feedback.  I changed it back to the normal text, but made the "ON" bolded, removed italics because they made the text harder to read, removed the colon because green and purple clashed, and aligned the message to look a lot better (IMO) since the spacing aligns.  Attaching screenshot example.
![new_safety_off_on](https://user-images.githubusercontent.com/4023541/232664588-82b4eecf-9c12-48d5-a192-b54e9126bfe9.png)